libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -496,11 +488,15 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       canNavigateForward: this.canNavigateForward,
       year: this.activeYear,
       monthName: this.activeMonthName,
-      weekDays: this.weekDays,
-      month: this.month,
+      weekDays: this._weekDays,
+      month: this._month,
     };
   }
 
+  private subtractTimezoneOffset(date: Date): Date {
+    return new Date(date.getTime() - date.getTimezoneOffset() * 60 * 1000);
```

Comment:
Consider readability:
```suggestion
    const timezoneOffsetInMs = date.getTimezoneOffset() * 60 * 1000;
    return new Date(date.getTime() - timezoneOffsetInMs);
```